core/src/main/java/io/grpc/inprocess/InProcessServerBuilder.java

```diff
@@ -76,6 +78,13 @@ public static InProcessServerBuilder forPort(int port) {
     throw new UnsupportedOperationException("call forName() instead");
   }
 
+  /**
+   * Generates a random and unique server name.
```

Comment:
Don't include "a random". This should be an okay implementation: "generated-name-" + atomic.getAndIncrement()